docs/dev/core/internal/payload.md

```diff
@@ -59,10 +59,18 @@ A [String](../../../book/reference/metrics/string.md) is represented by its stri
 
 A [String List](../../../book/reference/metrics/string_list.md) is represented as an array of strings.
 
+#### Example
+
 ```json
 ["sample string", "another one"]
 ```
 
+An empty list is accepted and send as one:
```

Comment:
```suggestion
An empty list is accepted and sent as:
```

Reply:
oh oops, fixing that in main.

Reply:
Ahh,  I didn't see you had the auto-merge set or I would have reserved my r+ to give you an opportunity to fix it